README.md

```diff
@@ -0,0 +1,237 @@
+```
+
+### Phase 2: GenTx
+#### (O) steps: 
+1. Copies `genesis.json` from (F) to local `.celestia-app/config/` directory:
+```sh
+cp <path_to_downloaded_genesis.json>/genesis.json  ~/.celestia-app/config/
+```
+2. Executes `gentx` command in CLI:
+```sh
+celestia-appd gentx $node_name 5000000000celes --keyring-backend=test --chain-id devnet-2
+``` 
+3. Passes `gentx-<hash>.json` file to (F). Location of the file can be find in this default path: 
+```sh
+cd .celestia-app/config/gentx/ && ls
+```
+#### (F) Steps:
+1. Copies `gentx-<hash>.json` from (O) to local `.celestia-app/config/gentx` directory
+2. Executes command in CLI:
+```sh
+celestia-appd collect-gentxs
+```
+3. Send the updated version of `genesis.json` to (O)
+
+### Phase 3: Launch
+#### (O) steps:
+1. Copies <b><i>updated</i></b> `genesis.json` from (F) to local `.celestia-app/config/` directory:
+```sh
+cp <path_to_downloaded_genesis.json>/genesis.json  ~/.celestia-app/config/
+```
+#### Everybody:  
+```sh
+celestia-appd start
+```
+
+### Verifying Celestia App Network
+
+To check that all validators are communicating with each other correctly, please look for a same block height for each of a running validator (let’s say block height = 100) 
+
+To confirm that everything is ok, you have to check that commit hashes for same block height are the same for all validators.
+
+### Phase X: Be a Validator in the running `devnet-2` network:
+1. Creates an account using a script in networks repository
+```sh
+cd networks/scripts
+node_name="chani"
+./1_create_key.sh $node_name
+```
+2. Asks any running validator to add funds to the account's address (ideally this should be changed with the upcoming faucet) in [Discord Developer Channel](https://discord.com/invite/YsnTPcSfWQ)
+3. Executes command in CLI:
+
+```sh
+celestia-appd init <moniker_name> --chain-id devnet-2
+```
+Example: 
+```sh 
+celestia-appd init fremen --chain-id devnet-2
+```
+4. Copies `genesis.json` from `networks/devnet-2` repo to local `.celestia-app/config/` directory:
+```sh
+cp networks/devnet-2/genesis.json  ~/.celestia-app/config/
+```
+5. Starts syncing the chain using command in CLI:
+```sh 
+celestia-appd start --p2p.seeds 2fd76fae32f587eceb266dce19053b20fce4e846@207.154.220.138:26656
+```
+Syncing finishes around 1-2 hours
+
+6. Executes command in CLI:
+```sh
+celestia-appd tx staking create-validator \
+ --amount=1000000000celes \
+ --pubkey=$(celestia-appd tendermint show-validator) \
+ --moniker=space \
+ --chain-id=devnet-2 \
+ --commission-rate=0.1 \
+ --commission-max-rate=0.2 \
+ --commission-max-change-rate=0.01 \
+ --min-self-delegation=1000000000 \
+ --from=$node_name \
+ --keyring-backend=test
+```
+
+## Running a Celestia Node
+### Pre-Requisites
+You need to have the trusted hash in order to initialize the full celestia node
+In order to know the hash, you need to query the celestia-app:
+```sh
+curl -s http://localhost:26657/block?height=1 | grep -A1 block_id | grep hash
```

Comment:
Usually this hash should be recent. Inside the unbonding period (instead of height=1). So this will fail without further context once this is addressed: https://github.com/celestiaorg/celestia-node/issues/205
It's OK for now but sth to keep in mind. Better would be to state that you should get a recent header (than this will always work).

Reply:
This is just an example of the query 😄 
But you are right, I'll explicitly put this comment in the doc once https://github.com/celestiaorg/celestia-node/issues/205 is done